### PR TITLE
Remove usage of HttpContextAccessor in CorsPolicyService

### PR DESCRIPTION
**Pull request by stefannikolei**
#56

### CODE REVIEW

File: src/EntityFramework/Services/CorsPolicyService.cs [hunk 1]
```diff
@@ -44,13 +44,12 @@ public async Task<bool> IsOriginAllowedAsync(string origin)
         {
             origin = origin.ToLowerInvariant();
 
-            // doing this here and not in the ctor because: https://github.com/aspnet/CORS/issues/105
```
Comment (brockallen) [inline src/EntityFramework/Services/CorsPolicyService.cs:47]: Can you re-add this comment.

**Review by brockallen — CHANGES_REQUESTED** [review]
Thanks! Just had one request in the PR. Please have a look.

**Comment by leastprivilege** [comment]
thanks!

**Comment by stefannikolei** [comment]
> thanks!

Should i look at other places where the services are resolved like that?

**Comment by leastprivilege** [comment]
Sure - if you like!

**Pull request merged**